### DISCUSSION TITLE
Test summary - Tag's used should be shown, and other tests hidden

### DISCUSSION THREAD

**Discussion by magnusjak** [maester365/maester#557] [Ideas]
The test below used the tag 'MS.EXO'. I want all tags and tag-exclusions shown in the top of the test summary.
Also, all other tests should not be shown as 'Not tested' and 'Total tests'. The MS.EXO tag contains 16 tests as of now, so I would expect the 'Total tests' info to only show 16, not 139. 
Or another solution to it could be to differentiate between 'Not tested' based on if they were just excluded with tagging, or if they were actually skipped because of something missing in the tenant etc. So in this scenario a 'skipped' info would show 2, and the 'Not tested' would show 123

![image](https://github.com/user-attachments/assets/aadcbd90-0a03-4d63-b651-33a2b9362484)

Could this be added? Using the -Path parameter correctly reduces the number of 'Total tests' in the summary, but tags are still not shown in any way.

**Comment by joshuafdo** [comment]
Thanks @magnusjak we reviewed this and it is a valid ask. Flagging as an enhancement for the backlog.

Thank you for your patience.
